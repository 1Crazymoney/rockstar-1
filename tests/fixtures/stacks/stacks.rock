Jazz is 123
Rock jazz
Shout jazz  (jazz is now a one-element array, so should output 1)
Shout jazz at 0 (should output 12345)
Shout jazz at 1 (should output mysterious)

