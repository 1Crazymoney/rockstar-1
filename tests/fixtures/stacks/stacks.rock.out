1
123
mysterious
